releasenotes/notes/remove-parametric-pulse-740cf905e96d6b82.yaml

```diff
@@ -0,0 +1,17 @@
+---
+upgrade:
+  - |
+    The ``ParametricPulse`` base class and pulse library were removed (previously located
+    at ``qiskit.pulse.library.parametric_pulses``), following their deprecation on
+    Qiskit 0.46.
+
+    The ``ParametricPulse`` base class and pulse library were superseded by the
+    :class:`~qiskit.pulse.SymbolicPulse` class and pulse library, which provide
+    better performance and QPY support. All removed library pulses have a
```

Comment:
I'm not sure if symbolic will provide better performance. Visualization is usually slow due to symbolic expression evaluation overhead.